### ISSUE TITLE
[P3] Replace Moment.js with Day.js

### ISSUE THREAD

**Issue by yoganlava** [open-duelyst/duelyst#254]
## Summary
As you probably know Moment.js has a fat bundle size (67.9kb). Day.js achieves (mostly) the same functionality while only being 2kb in size.

Might do this on my own as a first issue

**Comment by willroberts** [comment]
Good idea! You can also check out this issue for more Moment.js-related pain points: https://github.com/open-duelyst/duelyst/issues/52
